platform/security/platform-security-core-api/src/main/java/ddf/security/common/audit/SecurityLogger.java

```diff
@@ -42,17 +46,22 @@ private SecurityLogger() {
     }
 
     private static String getUser(Subject subject) {
-        String user;
-        try {
-            if (subject == null) {
-                subject = SecurityUtils.getSubject();
+        if (subject == null) {
+            subject = ThreadContext.getSubject();
+        }
+        if (subject == null) {
+            javax.security.auth.Subject javaSubject = javax.security.auth.Subject.getSubject(
+                    AccessController.getContext());
+            if(javaSubject!=null) {
```

Comment:
Not sure if you ran formatter on the entire section but the `userPrincipal!=null` looks a little off as well.